navigation-timing/nav2_test_response_end_and_duration_before_during_and_after_load_event.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE HTML>
+<html>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<iframe src="resources/respond_slowly.py"></iframe>
+<body>
+<script>
+async_test(function(t) {
+  window.addEventListener('message', t.step_func_done(function(event) {
```

Comment:
I typically find [`fetch_tests_from_window`](https://web-platform-tests.org/writing-tests/testharness-api.html#consolidating-tests-from-other-documents) to be an easier and more readable way to run tests in an iframe. ([usage example](https://source.chromium.org/chromium/chromium/src/+/main:third_party/blink/web_tests/external/wpt/resource-timing/resource-reload-TAO.html;l=15?q=fetch_tests_from_window%20f:wpt%20f:resource-timing&ss=chromium))

Reply:
I couldn't get it working.  This is based on the same pattern in navigation-timing/test_performance_attributes.sub.html

Reply:
OK, I guess we could clean up both at a later point (similar to the cleanups we're now doing for Resource Timing)